### DISCUSSION TITLE
Why not Windows?

### DISCUSSION THREAD

**Discussion by frkr** [sevdokimov/FriendlyTerminal#4] [Ideas]
cuz, has gitbash and wsl .... ?

**Comment by sevdokimov** [comment]
Not implemented yet, will be supported later.

**Reply by sevdokimov** [reply]
@frkr 1.1.0 is released, it supports WSL in Windows.  Appreciate any feedback.

**Reply by frkr** [reply]
I did several tests locally and over SSH and it all worked out great. It's really useful! It seems it didn't work with Github Copilot though. Hehe...I'm gonna do a 5-star review on Jetbrains.

**Reply by writzx** [reply]
any plans to support gitbash/cygwin for windows?
